### ISSUE TITLE
Get Jason-3_2_UNCOUPLED into git

### ISSUE THREAD

**Issue by mathomp4** [GEOS-ESM/GEOSgcm#30]
This one first depends on #29 first getting in and merged into `master`. We can then branch off `master` and commit the code from the `Jason-2_2` versions of:
```
LANL_Shared
GEOSsaltwater_GridComp
GEOSogcm_GridComp
```
We also need to *remove* `GEOSdataatm_GridComp` and get the `Jason-3_2_Modified_for_UNCOUPLED` variant of `GEOSgcm_GridComp.F90`.

Note that `CMakeLists.txt` files might/will probably need to adjusted because of the loss of `GEOSdataatm_GridComp` and perhaps added/removed files in `LANL_Shared` (not sure).

**Comment by mathomp4** [comment]
Closing for now as we may not want the uncoupled version on git until testing is done.
